ui/package.json

```diff
@@ -86,21 +85,23 @@
     "@testing-library/jest-dom": "^6.2.0",
```

Comment:
do you also want to update jest-dom to 6.4.0?

Reply:
good idea, thanks.  done.